from charles.exceptions.CustomException import CustomException

class LogicalUnitException(CustomException):
   """Raised when there is an issue with LogicalUnit """
   pass